service/src/main/java/tds/config/repositories/ConfigRepository.java

```diff
@@ -2,8 +2,11 @@
 
 import tds.config.ClientSystemFlag;
```

Comment:
General comment about this class.  I think TimeLimits and Client property stuff should be in separate repositories.  They are separate entity types and do not share anything other than the DB they hit. 

My concern with this broadly named interface/repository is that every call to the Configs DB will end up in this repository (and same goes for the service/controller).  You can already see the code growth in the tests.